recipes-bsp/tegra-binaries/tegra-helper-scripts/make-sdcard.sh

```diff
@@ -141,6 +141,27 @@ make_partitions() {
     return 0
 }
 
+create_filesystems() {
+    local partnumber
+    local pline mke2fscmd fstype
+    local errlog=$(mktemp)
+    for pline in "${PARTS[@]}"; do
+	eval "$pline"
+	if [ -n "$fstype" ] && [ "$fstype" != "basic" ]; then
```

Comment:
You should probably also check for `[ -z "$partfile" ]` here, since it doesn't make sense to format a partition if it's just going to get written over with a filesystem image later.